plugins/Morpheus/stylesheets/main.less

```diff
@@ -392,15 +496,19 @@ table.dataTable {
                       border-top: 5px solid @theme-color-brand;
                     }
 
+                    */
```

Comment:
it's there on purpose for now since the arrows look different in phantomjs (and only there) so I'm trying different solutions currently